msteams-platform/graph-api/rsc/preapproval-instruction-docs.md

```diff
@@ -0,0 +1,204 @@
+
+You can use the `SpecificSensitivityLabel` arguments to define specific sensitivity labels to apply the RSC preapproval policy to. 
+
+The following are an example of cmdlets that use these arguments:
+
+```powershell
+New-MgBetaTeamAppPreapproval -TeamsAppId c626ce8b-6d15-4c07-bfb1-a5fd0bc3c20e -ResourceSpecificApplicationPermissionsAllowedForTeams @('ChannelMessage.Read.Group') -TeamLevelSensitivityLabelCondition SpecificSensitivityLabel -SpecificSensitivityLabelIdsApplicableToTeams @('4de11089-adb9-4be8-9b7a-8336be68f3c4')
+```
+
+### Change RSC configuration to recognize explicitly defined preapproval policies
+
+After creating your preapproval policies, change your organization's RSC settings to use the new policy. This setting is more restrictive and might cause some apps not to function for your end users. To make this change, change the state of your organization's RSC settings. The following example shows the required PowerShell cmdlets:
+
+```powershell
+`Set-MgBetaTeamRscConfiguration -State ApprovedForPreApprovedAppsOnly`
+`Set-MgBetaChatRscConfiguration -State ApprovedForPreApprovedAppsOnly`
+```
+
+## Manage the existing preapproval policies
+
+After you create a preapproval policy, you can modify the policy to change the permissions and the sensitivity label of the policy. When apps are released with additional permissions, you must update the preapproval for that app to add the new permissions, ensuring that the app can be installed in your tenant. You need the same information to manage an existing preapproval and create a new preapproval. 
+
+You can also delete an existing preapproval policy if you want to stop preapproving an existing app’s RSC permission.
+
+### Update an existing preapproval policy
+
+You can update a preapproval policy with the `Update-MgBetaTeamAppPreApproval` cmdlet. When you specify the permissions to be updated, you must distinguish the type of RSC permissions.
+
+The following RSC configurations are available to manage an existing preapproval:
+
+|Configuration| Description|
+|---|---|
+|`ResourceSpecificApplicationPermissionsAllowedForChats`| Use this configuration when you create a preapproval for RSC permissions for a chat. For the list of all permissions, see [RSC permissions](/graph-api/rsc/resource-specific-consent). |
+|`ResourceSpecificApplicationPermissionsAllowedForTeams`| Use this configuration when you create a preapproval for RSC permissions for a team, use this configuration. For the list of all permissions, see [RSC permissions](/graph-api/rsc/resource-specific-consent).|
+
+#### Update the preapproval for chat RSC
+
+```powershell
+Update-MgBetaTeamAppPreapproval -TeamsAppId c626ce8b-6d15-4c07-bfb1-a5fd0bc3c20e -ResourceSpecificApplicationPermissionsAllowedForChats @('OnlineMeeting.ReadBasic.Chat', 'TeamsAppInstallation.Read.Chat ')
+```
+
+#### Update the preapproval for team RSC
```

Comment:
should be consistent. sometimes we are saying "preapproval" and sometimes "preapproval policy".

Reply:
changed all instances to be consistent, except those that refer to the preapproval settings.